src/views/Project/index.jsx

```diff
@@ -117,8 +133,20 @@ export default class Project extends Component {
             Bugs & Issues
           </Typography>
         </header>
-        <TasksTable items={data} />
+        {error && <ErrorPanel error={error} />}
+        {loading && <Spinner />}
+        {!error && !loading && <TasksTable items={data} />}
       </Fragment>
     );
   }
 }
+
+const ProjectClient = props => (
+  <Query
+    query={gql`{${githubQuery(props.match.params.project)}}\n${Issues}`}
+    skip={!projects[props.match.params.project].repositories}>
+    {githubData => <Project match={props.match} githubData={githubData} />}
```

Comment:
Remove the `match` props. If you're trying to have access to `props.match`, then you can use the `withRouter` decorator similar to how it's being done in https://github.com/taskcluster/taskcluster-web/blob/2998f6561a4de26908d4a2eba4aaca8443c15a53/src/components/RoleScopesTable/index.jsx#L34.